misc/docs/source/packages/obspy.io.nied.knet.rst

```diff
@@ -0,0 +1,14 @@
+.. currentmodule:: obspy.io.nied.knet
+.. automodule:: obspy.io.nied.knet
+
+    .. comment to end block
+
+    Modules
+    -------
+    .. autosummary::
+       :toctree: autogen
+       :nosignatures:
+
+       core
```

Comment:
There is no `core` within `obspy.io.nied.knet`. In fact, I'm not sure you even need this file any more.